### PR TITLE
Fix extra scrollbar on log container, auto scroll logs to bottom

### LINKED ISSUE #774: [BUG] Log "direction" is oldest first
**Describe the bug**
The "direction" of the log in the UI is oldest first. I think I remember this as being newest first, which I find better, as no scrolling is needed to see recent entries.

I labeled this as a bug since I believe this is a regression.

### PR DESCRIPTION
**Pull request by shamoon**
This PR closes #774 and #775 , it:

- Fixes css that caused an extra scrollbar on the log container
- ~~Reverses the order of log output (newest first)~~
- Automatically scrolls logs to the bottom

### CODE REVIEW

**Comment by jonaswinkler** [comment]
> * Reverses the order of log output (newest first)

No. The log may also contain stack traces when critical errors occur, and these are unreadable when reversed. Pretty much all logfiles have the newest entry on the bottom.

**Comment by shamoon** [comment]
> > * Reverses the order of log output (newest first)
> 
> No. The log may also contain stack traces when critical errors occur, and these are unreadable when reversed. Pretty much all logfiles have the newest entry on the bottom.

Unreadable as in they appear in reverse order and thus hard to understand? Ok I can revert that part of this PR, shoulda waited for your reply...

**Comment by jonaswinkler** [comment]
> Unreadable as in they appear in reverse order and thus hard to understand?

If anything, make it scroll to the bottom automatically. Github actions does this.

**Comment by shamoon** [comment]
> > Unreadable as in they appear in reverse order and thus hard to understand?
> 
> Exactly. If anything, make it scroll to the bottom automatically. Github actions does this.

Ok let me see what I can do with that

**Comment by shamoon** [comment]
Ok check this out LMK

File: src-ui/src/app/components/manage/logs/logs.component.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { Component, ElementRef, AfterViewChecked, ViewChild } from '@angular/core';
 import { LogService } from 'src/app/services/rest/log.service';
 
 @Component({
   selector: 'app-logs',
   templateUrl: './logs.component.html',
   styleUrls: ['./logs.component.scss']
 })
-export class LogsComponent implements OnInit {
+export class LogsComponent implements AfterViewChecked {
```
Comment (jonaswinkler) [inline src-ui/src/app/components/manage/logs/logs.component.ts:9]: Don't remove OnInit, this has a `ngOnInit` method.

File: src-ui/src/app/components/manage/logs/logs.component.ts [hunk 2]
```diff
@@ -26,9 +28,14 @@ export class LogsComponent implements OnInit {
...
   }
 
+  ngAfterViewChecked() {
+    this.scrollToBottom();
+  }
+
   reloadLogs() {
     this.logService.get(this.activeLog).subscribe(result => {
       this.logs = result
+      this.scrollToBottom()
```
Comment (jonaswinkler) [inline src-ui/src/app/components/manage/logs/logs.component.ts:38]: Is this necessary? Works even without that.

**Review by jonaswinkler — CHANGES_REQUESTED** [review]

File: src-ui/src/app/components/manage/logs/logs.component.ts (hunk 2 above)
Comment (shamoon) [reply src-ui/src/app/components/manage/logs/logs.component.ts:38]: Doh, yea

**Pull request merged**